test/stream-monitor/stream_sources/amqp_source.py

```diff
@@ -215,13 +215,22 @@ def __setup_generated_amqp(self, generate_string):
             fixed.sendline_and_stat(r'''rabbitmqctl set_permissions {} ".*" ".*" ".*"'''.format(auser), must_be_0=True)
             fixed.logout()
             return 'amqp://{}:{}@{}:{}'.format(auser, apw, fixed.dut_ssh_host, port), auser
-        except ExceptionPxssh as ex:
+        except EOF as ex:
             # the PR in-queue behind with one sets up better logging in the sources.
             # Rather than try to pull that forward, manually grab flogging and
             # update this block in next merge.
             from flogging import get_loggers
             logs = get_loggers()
-            logs.irl.debug('unable to set up amqp user. AMQP monitors disabled: %s', ex)
+            logs.irl.warning('unable to connect ot instance to setup AMQP user. AMQP monitors disabled: %s', ex)
```

Comment:
Typo:  ot -> to
I won't hold up for the typo but we should probably fix this in a soon-to-follow PR since it is user visible.